Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/MainDemoTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Template.NUnit.Constants;
+using Aquality.Selenium.Template.NUnit.Steps;
+using Aquality.Selenium.Template.Utilities;
+using NUnit.Framework;
+
+namespace Aquality.Selenium.Template.NUnit.Tests
+{
+    public class MainDemoTest : BaseTest
+    {
+        private readonly MainPageSteps mainPageSteps = new MainPageSteps();
+
+        [SetUp]
+        public new void Setup()
+        {
+            GoToPageStartPage();
+            SetScreenExpansionMaximize();
+        }
+
+        [TearDown]
+        public new void AfterEach()
+        {
+            AqualityServices.Browser.Quit();
+        }
```

Comment:
This should be done in the BaseTest class, and within the if(AqualityServices.IsBrowserStarted)

Reply:
This code was removed.